recipe/meta.yaml

```
@@ -11,16 +10,16 @@ source:
   sha256: 75b1c665699ec0f1ffce1ba3d776f7dfce802156f22e70a7b9c8f0b4d7e80f42
 
 build:
-  skip: true  # [py<36]
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  # maturin isn't availabe on s390x
+  skip: true  # [py<36 or s390x]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('c') }}
-    - {{ compiler('rust') }}
+    - {{ compiler('rust') }}  # [not win]
```

```suggestion
    - {{ compiler('rust') }}
```

In conda_build_config.yaml, two rust compilers are configured, rust and rust-gnu, resulting in two variants.
It looks like you had an issue with rust-gnu (possibly from having compiler('c') set).
If you only want to build the rust variant, it is better to skip rust-gnu with a selector.